packages/plugin-eslint/package.json

```diff
@@ -3,8 +3,16 @@
   "version": "0.1.0",
   "dependencies": {
     "@code-pushup/utils": "*",
+    "@code-pushup/models": "*",
     "eslint": "~8.46.0",
-    "zod": "^3.22.4",
-    "@code-pushup/models": "*"
+    "zod": "^3.22.4"
+  },
+  "peerDependencies": {
+    "@nx/devkit": "^17.0.0"
+  },
+  "peerDependenciesMeta": {
+    "@nx/devkit": {
+      "optional": true
```

Comment:
@nx/devkit can not be optional. Trying to import something from the `plugin-eslint` package will throw an error like "module not found". To fix it, the package has to have an additional entry point like `@code-pushup/eslint-plugin/nx`, or `nx/devkit` has to be imported lazy.